### ISSUE TITLE
TAB: Disable TAB Auto Refresh (Reload)

### ISSUE THREAD

**Issue by ghost** [brave/brave-browser#41469]
### Description

It happens on any site, switching to another TAB for a minute and then switching back again will automatically Refresh (Reload) the TAB.

### Steps to reproduce

1. https://github.com/brave/brave-browser/issues
2. Creating issues. Editing. Entering some text.
3. Switch to other TAB 1 minute.
4. Switch back to the issues being edited TAB.
5. Auto Refresh (Reload) has occurred, the contents of the issues being edited are gone.

### Actual result

No automatic Refresh (Reload)

### Expected result

Easily reproduced

### Reproduces how often

Easily reproduced

### Brave version

1.70.123 Chromium: 129.0.6668.89 (Official Build) (64-bit) 

### Device

- Brand/model: ANY
- Android version: ANY

### Channel information

- [x] release (stable)
- [x] beta
- [x] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by bugsbugsbux** [comment]
yes, i hate this, tho i takes a bit longer for me until it refreshes. happens since more than a year now. watch a video, switch tab or minimize browser for like 15min and the site refreshes losing the playback position; same when on a page which doesnt put queries into the url: leave for a bit and the page loses its state. if my internet does not work, i cant even read the page i already loaded earlier, because the stupod site reloads and tells me "no connection"

android, Brave 1.71.118, Chromium 130.0.6723.70

**Comment by codey-badger** [comment]
It gets me out of flow. 

Is there any workaround?

**Comment by damrath** [comment]
Can confirm is still an issue on

Version 1.91.175
com.brave.browser
versionCode 429117504

I've been filling out a form on a website, briefly went to home screen, clicked the brave search bar to search something, swipe back to the original tab by swiping on brave's menu bar and the website reloads for no apparent reason, clearing all my forms in the process
